app/src/main/res/layout/profile_chooser_profile_view.xml

```diff
@@ -81,7 +81,7 @@
           android:id="@+id/profile_is_admin_text"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:layout_marginTop="@dimen/profile_chooser_profile_view_is_admin_margin_top"
+          android:layout_marginTop="@dimen/profile_chooser_profile_text_view_layout_margin_top"
```

Comment:
@rt4914 sir can you pls review this part as I m having some doubt regarding this way of change

Reply:
@Rishavgupta12345 Please keep this previous name as it provides context about when to use to this margin
`profile_chooser_profile_view_is_admin_margin_top`